akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala

```diff
@@ -61,7 +61,7 @@ object FileIO {
    * @param options File open options
    */
   @deprecated("Use `toPath` instead.", "2.4.5")
-  def toFile(f: File, options: util.Set[StandardOpenOption]): javadsl.Sink[ByteString, CompletionStage[IOResult]] =
+  def toFile[Opt <: OpenOption](f: File, options: util.Set[Opt]): javadsl.Sink[ByteString, CompletionStage[IOResult]] =
```

Comment:
Is that signature really what we want for the Java side? Shouldn't it be `util.Set[_ <: OpenOption]`?